pom.xml

```diff
@@ -85,7 +85,7 @@
         <version.org.jboss.ws.jbossws-spi>3.3.0.Final</version.org.jboss.ws.jbossws-spi>
         <version.org.kohsuke.metainf-services.metainf-services>1.7</version.org.kohsuke.metainf-services.metainf-services>
         <version.junit.junit>4.13.1</version.junit.junit>
-        <version.jmockit>1.33</version.jmockit>
+        <version.jmockit>1.34</version.jmockit>
```

Comment:
Thanks very much for working on this @Ashpan! Nice to see Ubuntu runs passing again!

Just one small comment. It would be good to create a separate ELY issue for the jmockit component upgrade and include that as a second commit here.